anyric/super-record
# #168629857 Add Report Analytics Graphs

#### What does this PR do?
This PR add graphs and charts analytic reports on the Dashboard

#### Description of Task to be completed?
- Add graphs and charts reports on the Dashboard

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to login
- You should see analytics graphs on the Dashboard

#### Any background context you want to provide?
Currently there are no graphical analysis of reports for quick and easier interpretations. There should be graphs and charts created and displayed on the dashboard.

#### What are the relevant pivotal tracker stories?
[#168629857](https://www.pivotaltracker.com/story/show/168629857)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/65425571-f4401d00-de16-11e9-914b-a38f3ba3386e.png)


#### Questions:
N/A
